google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/DatasetInfo.java

```diff
@@ -132,6 +133,20 @@ public Dataset apply(DatasetInfo datasetInfo) {
     public abstract Builder setDefaultEncryptionConfiguration(
         EncryptionConfiguration defaultEncryptionConfiguration);
 
+    /**
+     * [Optional] The default partition expiration for all partitioned tables in the dataset, in
+     * milliseconds. Once this property is set, all newly-created partitioned tables in the dataset
+     * will have an expirationMs property in the timePartitioning settings set to this value, and
+     * changing the value will only affect new tables, not existing ones. The storage in a partition
```

Comment:
will only --> only

(Tech writing lives in the eternal present)

Reply:
Done.